cmd/models/list_test.go

```diff
@@ -96,13 +96,13 @@ func TestListModelsSinglePage(t *testing.T) {
 
 	response := openfga.ReadAuthorizationModelsResponse{
 		AuthorizationModels: &models,
-		ContinuationToken:   nil,
+		ContinuationToken:   openfga.PtrString(""),
 	}
 	mockExecute.EXPECT().Execute().Return(&response, nil)
 
 	mockRequest := mockclient.NewMockSdkClientReadAuthorizationModelsRequestInterface(mockCtrl)
 	options := client.ClientReadAuthorizationModelsOptions{
-		ContinuationToken: nil,
+		ContinuationToken: openfga.PtrString(""),
```

Comment:
What happens if this is nil?

Reply:
For the unit test, we are "expecting" empty string.  However, for the SDK itself, passing in nil should still be fine.